backend/apps/images/main.py

```diff
@@ -32,11 +32,15 @@
 from config import (
     SRC_LOG_LEVELS,
     CACHE_DIR,
+    IMAGES_GENERATION_ENGINE,
     ENABLE_IMAGE_GENERATION,
     AUTOMATIC1111_BASE_URL,
     COMFYUI_BASE_URL,
     IMAGES_OPENAI_API_BASE_URL,
     IMAGES_OPENAI_API_KEY,
+    IMAGES_MODEL,
```

Comment:
Should be more explicit `IMAGE_GENERATION_MODEL`

Reply:
ok,i will recompose it